src/account/account.entity.ts

```diff
@@ -63,6 +82,40 @@ export class Account extends BaseEntity {
         }
     }
 
+    get name(): string | null {
+        return this._name;
+    }
+
+    get bio(): string | null {
+        return this._bio;
+    }
+
+    get avatarUrl(): URL | null {
+        return this._avatarUrl;
+    }
+
+    get bannerImageUrl(): URL | null {
+        return this._bannerImageUrl;
+    }
+
+    updateProfile(params: ProfileUpdateParams): void {
```

Comment:
Do we need to add `username` here? Or maybe that's coming as part of other changes(custom handles)?